2016/presentations/wprowadzenie_do_GeoPythona/text.md

```diff
@@ -0,0 +1,134 @@
+#Krótkie wprowadzenie do GeoPythona – Małgorzata Papież
+
+Od kilku lat, w Polsce coraz większą popularność zdobywają aplikacje oparte na wykorzystaniu *danych przestrzennych*. 
+To dzięki nim przestaliśmy być zależni od papierowych map i bez wychodzenia z domu możemy zobaczyć każdy zakątek Ziemi. 
+Co jeśli dostępne aplikacje takie jak Google Earth, Google Maps nam nie wystarczają i chcemy czegoś więcej ? 
+Zobaczmy, więc jak wykorzystując Pythona możemy zaprojektować aplikacje pozwalające nam na tworzenie własnych map czy przeglądanie najnowszych zdjęć satelitarnych.
+
+## Dane przestrzenne 
+Rozwój technologii komputerowych w ostatnich latach sprawił, że przetwarzanie bardzo dużej ilości danych w czasie rzeczywistym stało się możliwe. Przyspieszenie analityki dużych zbiorów danych przyczyniło się wzrostu zainteresowania danymi przestrzennymi. Dane przestrzenne przechowują informacje o obiekcie świata rzeczywistego biorąc pod uwagę jego położenie przestrzenne. Oprócz informacji przestrzennej podanej w postaci szerokości i długości geograficznej mogą zawierać również inne istotne informacje takie jak data utworzenia, związki przestrzenne obiektów ze sobą tzw. topologię czy właściwości danego obiektu czyli jego atrybuty. Dane przestrzenne dzieli się na wektorowe i rastrowe. Pierwsze z nich reprezentowane są przez obiekty geometryczne takie jak punkty, linie, powierzchnie, których kształt i położenie definiowany jest przez współrzędne. Taki sposób reprezentacji używany jest z racji swojej dużej dokładności, możliwości wyodrębniania poszczególnych obiektów oraz małej pojemności plików. Do obiektów wektorowych w rzeczywistości należą granice działek, ulice, budynki, sieci: elektryczne, gazowe, wodne, telefoniczne itd. W modelu rastrowym dane są przechowywane w postaci pojedynczych pikseli, w regularnej siatce pikseli. O ile model wektorowy przeznaczony jest do przechowywania danych o charakterze dyskretnym, tak model rastrowy jest przypisany do danych o charakterze ciągłym. Dzięki temu znacznie wydajniej radzi sobie z analizą i modelowaniem zjawisk zachodzących w przestrzeni np. stanu zanieczyszczenia, opadów atmosferycznych. Niestety wymaga dużo pamięci RAM oraz powoduje utratę części informacji.
```

Comment:
> O ile model wektorowy przeznaczony jest do przechowywania danych o charakterze dyskretnym, tak model rastrowy jest przypisany do danych o charakterze ciągłym.

Nie do końca rozumiem to zdanie. Przecież raster to piksele, czy jak najbardziej dane nieciągłe. Chyba, że nie zrozumiałem znaczenia słowa "dyskretny" i chodzi o dane poufne?